site/content/docs/main/contributions/tencent-config.md

```diff
@@ -0,0 +1,184 @@
+
+- `--bucket`: The bucket name created at Tencent Cloud COS.
+
+- `--secret-file`: Access tencent cloud COS access credential file for the "credentials-velero" credential file created above.
+
+- `--use-restic`: Back up and restore persistent volume data using the open source free backup tool [restic](https://github.com/restic/restic).  However, 'hostPath' volumes are not supported, see the [restic limit](https://velero.io/docs/v1.5/restic/#limitations) for details), an integration 		that complements Velero's backup capabilities and is recommended to be turned on.
+
+- `--default-volumes-to-restic`: Enable the use of Restic to back up all Pod volumes, provided that the `--use-restic`parameter needs to be turned on.
+
+- `--backup-location-config`: Back up the bucket access-related configuration:
+
+  `region`: Tencent cloud COS bucket area, for example, if the created region is Guangzhou, the Region parameter value is "ap-guangzhou".
+
+  `s3ForcePathStyle`: Use the S3 file path format.
+
+  `s3Url`: Tencent Cloud COS-compatible S3 API access address,Note that instead of creating a COS bucket for public network access domain name, you must use a format of "https://cos.`region`.myqcloud.com" URL, for example, if the region is Guangzhou, the parameter value is "https://cos.ap-guangzhou.myqcloud.com.".
+
+There are other installation parameters that can be viewed using `velero install --help`, such as setting `--use-volume-snapshots-false`  to close the storage volume data snapshot backup if you do not want to back up the storage volume data.
+
+After executing the installation commands above, the installation process looks like this:
+
+{{< figure src="https://main.qcloudimg.com/raw/9015313121ed7987558c88081b052574.png" width="100%">}}
+
+After the installation command is complete, wait for the velero and restic workloads to be ready to see if the configured storage location is available.
+
+Executing the 'velero backup-location get' command to view the storage location status and display "Available" indicates that access to Tencent Cloud COS is OK, as shown in the following image:
+
+{{< figure src="https://main.qcloudimg.com/raw/69194157ccd5e377d1e7d914fd8c0336.png" width="100%">}}
+
+At this point, The installation using Tencent Cloud COS as Velero storage location is complete, If you need more installation information about Velero, You can see the official website [Velero documentation](https://velero.io/docs/) .
+
+## Velero backup and restore example
+
+In the cluster, use the helm tool to create a minio test service with a persistent volume, and the minio installation method can be found in the [minio installation](https://github.com/minio/charts), in which case I have bound a load balancer for the minio service to access the management page using a public address in the browser.
+
+{{< figure src="https://main.qcloudimg.com/raw/f0fff5228527edc72d6e71a50d5dc966.png" width="100%">}}
```

Comment:
Can you add these images to the velero repo?

Reply:
Thank you, but I have the following confusion:
1. Which directory should I put the picture in? /docs/main/img?
2. Since I need to customize the width of the picture, I use figure shortcodde, the local link makes it impossible for me to get to src. I don't know how to fix it.

Reply:
I have been solved the second  confusion，thank u very much.

Reply:
I also create a directory "contributions/img-for-tencent"  for put the images to solve the first  confusion.